src/transformers/utils/logging.py

```diff
@@ -101,7 +101,8 @@ def _configure_library_root_logger() -> None:
             formatter = logging.Formatter("[%(levelname)s|%(pathname)s:%(lineno)s] %(asctime)s >> %(message)s")
             _default_handler.setFormatter(formatter)
 
-        library_root_logger.propagate = False
+        is_ci = os.getenv("CI") is not None and os.getenv("CI").upper() in {"1", "ON", "YES", "TRUE"}
+        library_root_logger.propagate = True if is_ci else False
```

Comment:
We need `propagate = True` in order to get live log (here for `warnings`), but let's only do it when `CI` env. variable is set.
This variable is set on GitHub Actions and CircleCI. 

On CircleCI, we use `-n xxxx` (`xdist`) which won't show live log, which is preferred (to keep it more quite).
On GitHub Actions, we don't use `-n xxx` and live log is shown, so we can grab logs with warning info.